api/feature-flags.js

```
@@ -0,0 +1,41 @@
+  feedbackMechanism: true,
+  auditTrail: true,
+  mobileResponsiveness: true,
+};
+
+router.get('/feature-flags', (req, res) => {
+  res.json(featureFlags);
+});
+
+router.post('/feature-flags', (req, res) => {
+  const { feature, enabled, implementation } = req.body;
+  if (featureFlags.hasOwnProperty(feature)) {
```

Biome (1.9.4)

[error] Do not access Object.prototype method 'hasOwnProperty' from target object.

It's recommended using Object.hasOwn() instead of using Object.hasOwnProperty().
See MDN web docs for more details.

(lint/suspicious/noPrototypeBuiltins)